misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -3814,19 +3816,20 @@ protected <T extends Node> SeparatedNodeList<T> formatSeparatedNodeList(Separate
      * @param itemTrailingWS Number of single-length spaces to be added after each item in the list
      * @param itemTrailingNL Number of newlines to be added after each item in the list
      * @param separatorTrailingWS Number of single-length spaces to be added after each separator in the list
-     * @param separatorTrailingNL Number of newlines to be added after each each separator in the list
+     * @param separatorTrailingNL Number of newlines to be added after each separator in the list
      * @param listTrailingWS Number of single-length spaces to be added after the last item in the list
      * @param listTrailingNL Number of newlines to be added after the last item in the list
+     * @param allowInAndMultiLine Allow both inline and multiline formatting at the same time
      * @return Formatted node list
      */
-    @SuppressWarnings("unchecked")
     protected <T extends Node> SeparatedNodeList<T> formatSeparatedNodeList(SeparatedNodeList<T> nodeList,
-                                                                            int itemTrailingWS,
-                                                                            int itemTrailingNL,
-                                                                            int separatorTrailingWS,
-                                                                            int separatorTrailingNL,
-                                                                            int listTrailingWS,
-                                                                            int listTrailingNL) {
+                                                                                int itemTrailingWS,
+                                                                                int itemTrailingNL,
+                                                                                int separatorTrailingWS,
+                                                                                int separatorTrailingNL,
+                                                                                int listTrailingWS,
+                                                                                int listTrailingNL,
+                                                                                boolean allowInAndMultiLine) {
```

Comment:
Is this formatting intentional?

Reply:
No i's not, it is accidental. i'll revert it back